integration-tests/src/test/java/org/mvndaemon/mvnd/it/ResidentExtensionsTest.java

```diff
@@ -69,8 +69,8 @@ void verify() throws IOException, InterruptedException {
     }
 
     private void assertDaemonRegistrySize(int size) {
-        Assertions.assertThat(registry.getAll().size())
-                .as("Daemon registry size should be " + size)
-                .isEqualTo(size);
+        Assertions.assertThat(registry.getAll())
+                .as("Daemon registry size should be %d actually containing:(%s)", size, registry.getAll())
```

Comment:
not sure what "actually containing" means here. It's a tad awkward phrasing. Is this a UI message somewhere or just a test failure message?